docs/integrations.md

```diff
@@ -17,3 +17,41 @@ registering data producers and consumers, as well as any data transformations th
     pipelines with Aurelius Atlas using the Lineage API
 
 For further details and guidelines on usage, please visit the [API documentation](https://github.com/aureliusenterprise/aurelius/blob/main/backend/m4i-lineage-rest-api/README.md).
+
+## Federated Identity with Keycloak
```

Comment:
Same here, better to put the focus on corporate identity providers like e.g. Azure AD rather than social login. We can mention this as a secondary option.